src/wizard/javascript/angular.md

```diff
@@ -161,41 +257,44 @@ npm install @sentry/browser @sentry/integrations
 ```
 
 ```javascript
-import angular from 'angular';
-import * as Sentry from '@sentry/browser';
-import { Angular as AngularIntegration } from '@sentry/integrations';
+import angular from "angular";
+import * as Sentry from "@sentry/browser";
+import { Angular as AngularIntegration } from "@sentry/integrations";
 
 // Make sure to call Sentry.init after importing AngularJS.
 // You can also pass {angular: AngularInstance} to the Integrations.Angular() constructor.
 Sentry.init({
-  dsn: '___PUBLIC_DSN___',
-  integrations: [
-    new AngularIntegration(),
-  ],
+  dsn: "___PUBLIC_DSN___",
+  integrations: [new AngularIntegration()],
 });
 
 // Finally require ngSentry as a dependency in your application module.
-angular.module('yourApplicationModule', ['ngSentry']);
-
+angular.module("yourApplicationModule", ["ngSentry"]);
 ```
 
 In case you're using the CDN version or the Loader, Sentry provides a standalone file for every integration:
 
 ```html
 <!-- Note that we now also provide a es6 build only -->
 <!-- <script src="https://browser.sentry-cdn.com/5.20.1/bundle.es6.min.js" integrity="sha384-vX2xdItiRzNmed/VJFb8J4h2p35hYqkdTI9+xNOueKEcr7iipZy17fplNS0ikHL0" crossorigin="anonymous"></script> -->
-<script src="https://browser.sentry-cdn.com/5.20.1/bundle.min.js" integrity="sha384-O8HdAJg1h8RARFowXd2J/r5fIWuinSBtjhwQoPesfVILeXzGpJxvyY/77OaPPXUo" crossorigin="anonymous"></script>
+<script
+  src="https://browser.sentry-cdn.com/5.20.1/bundle.min.js"
+  integrity="sha384-O8HdAJg1h8RARFowXd2J/r5fIWuinSBtjhwQoPesfVILeXzGpJxvyY/77OaPPXUo"
+  crossorigin="anonymous"
+></script>
 
 <!-- If you include the integration it will be available under Sentry.Integrations.Angular -->
-<script src="https://browser.sentry-cdn.com/5.20.1/angular.min.js" crossorigin="anonymous"></script>
+<script
+  src="https://browser.sentry-cdn.com/5.20.1/angular.min.js"
+  crossorigin="anonymous"
+></script>
 
 <script>
   Sentry.init({
-    dsn: '___PUBLIC_DSN___',
-    integrations: [
-      new Sentry.Integrations.Angular(),
-    ],
+    dsn: "___PUBLIC_DSN___",
+    integrations: [new Sentry.Integrations.Angular()],
```

Comment:
Shouldn't `Angular` be a default integration when using `@sentry/angular`?

Reply:
It's an old Angular 1.x integration, not `@sentry/angular`.

Reply:
Aha! Got it.